Quadriga/src/main/webapp/WEB-INF/views/workbench/project.jsp

```diff
@@ -95,6 +95,7 @@
 			</div>
 		</c:forEach>
 
+		<c:if test="${editoraccess != '1' || owner == '1' || isProjectAdmin || isProjectContributor}">
```

Comment:
When the user has only the role of 'editor' which is indicated by 'editoraccess' here, he shouldn't see the collaborator's workspaces in the project details page. So that's the reason the check is being done here